build.sbt

```diff
@@ -3931,6 +3937,9 @@ lazy val `engine-runner` = project
             .listFiles("*.jar")
             .map(_.getAbsolutePath()) ++
           `std-tableau-polyglot-root`
+            .listFiles("*.jar")
```

Comment:
- I would really prefer if we could go without these three lines 
- it is an excellent opportunity to make _dual JVM mode_ work smoothly
- I promise to address any inquiries ASAP
- btw. try the experience execute:

```
sbt:enso> runEngineDistribution --run test/DuckDB_Tests --vm.D=polyglot.enso.classLoading=Standard.DuckDB:guest,enso_dev.DuckDB_Test:guest,hosted
```
- does it work? Yes, then let's try it. No, tell me what's failing!

Reply:
Test code:
```
from Standard.Base import all
from Standard.Table import all
from Standard.Database import all
from Standard.DuckDB import all

main =
    connection5 = Database.connect DuckDB.In_Memory
    table4 = connection5.execute_query 'SELECT * FROM "/mnt/c/Users/jdunk/Downloads/flights-1m.parquet"' ..All_Rows write_operation=False
    any10 = table4.aggregate columns=[..Count, ..Maximum 'ARR_DELAY']
    any10.display
```

Reply:
Data file: https://www.agentsfordata.com/sample-datasets/flights-1m.parquet